src/components/forms/formr-part-b/viewSections/ViewSectionTestHelper.tsx

```diff
@@ -0,0 +1,112 @@
+import { mount } from "@cypress/react";
+import {
+  FormRPartB,
+  Work,
+  Declaration,
+  CovidDeclaration
+} from "../../../../models/FormRPartB";
+import { DateUtilities } from "../../../../utilities/DateUtilities";
+import { BooleanUtilities } from "../../../../utilities/BooleanUtilities";
+
+type FormRPartBField = keyof FormRPartB;
+type FormRPartBWorkField = keyof Work;
+type FormRPartBDeclarationField = keyof Declaration;
+type FormRPartBCovidDeclarationField = keyof CovidDeclaration;
+
+export interface ISectionDataField {
+  fieldName: FormRPartBField;
+  format: string;
+}
+
+export interface ISectionWorkDataField {
+  fieldName: FormRPartBWorkField;
+  format: string;
+}
+
+export interface ISectionDeclarationDataField {
+  fieldName: FormRPartBDeclarationField;
+  format: string;
+}
+
+export interface ISectionCovidDeclarationDataField {
+  fieldName: FormRPartBCovidDeclarationField;
+  format: string;
+}
+
+function CheckDataIsDisplayed(dataValue: any, format: string) {
```

Comment:
Unsure of the benefits of defining this as` const CheckDataIsDisplayed = (dataValue: any, format: string) => {` (or something else more idiomatic for React!)

Also: naming conventions for this file and its contents?

Reply:
@ReubenRobertsHEE, only advantage I've found with using `function` syntax is when you've got a file with your functions chaotically ordered and JS saves your bacon by hoisting them! 🤣 As far a I remember there's also some difference in the way `this` is scoped too. 
As far as naming conventions go, I wonder if we can make those `ViewSection...ShouldIncludeThisData` functions into one generic function to test our TS skills to the limit! 😅

Reply:
That would be super-cool, the repetition is a bit 💩 . I might need a bit of help on that (or time to go a-digging, but that might be any amount of time!)